flake.nix

```diff
@@ -81,6 +81,7 @@
         dell-poweredge-r7515 = import ./dell/poweredge/r7515;
         dell-precision-3541 = import ./dell/precision/3541;
         dell-precision-5490 = import ./dell/precision/5490;
+        dell-precision-5520 = import ./dell/precision/5520;
```

Comment:
Please also add this model to the README.md table. Otherwise our CI doesn't test this.